docs/api_extra.rst

```diff
@@ -904,3 +904,43 @@ converting to Python.
     number of seconds, and fractional seconds are supported to the
     extent representable. The resulting timepoint will be that many
     seconds after the target clock's epoch time.
+
+
+Evaluating Python expressions from strings
+------------------------------------------
+
+The following functions can be used to evaluate Python functions and
+expressions. They require an additional include directive:
+
+.. code-block:: cpp
+
+   #include <nanobind/eval.h>
+
+Detailed documentation including example code is provided in a :ref:`separate
+section <eval>`.
+
+.. cpp:enum-class:: eval_mode
+
+   This enumeration characterizes the start symbol for different kinds of
```

Comment:
"start symbol" -- really (maybe in a Chomsky grammar kind of way)? This is likely confusing to users, I would say: "This enumeration specifies how the content of a string should be interpreted."

Reply:
Took the expression from [here](https://docs.python.org/3/c-api/veryhigh.html#c.Py_eval_input). Will change as suggested.